STATUS: INLINE FAIL SETUPTEARDOWN
MODIFIED: de.strullerbaumann.visualee.ui.graph.boundary.GraphCreatorTest.testBuildJSONLinks
MODIFIED FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/ui/graph/boundary/GraphCreatorTest.java
CLEANER: de.strullerbaumann.visualee.source.entity.JavaSourceFactoryTest.testGetInstance
CLEANER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/source/entity/JavaSourceFactoryTest.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.PackageFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/PackageFilterTest.java
ORIGINAL CLEANER SIZE: 5
NEW CLEANER SIZE: 1
ELAPSED TIME: 28.985999822616577
==========================
@@ -82,0 +82,3 @@
+ {
+     FilterContainer.getInstance().clear();
+ }
